### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
         "en": "Hosting platform made for podcasters",
         "fr": "Plateforme d'hébergement conçue pour les podcasteurs"
     },
-    "version": "1.0.0-80~ynh1",
-    "url": "https://podlibre.org/",
+    "version": "1.0.0-5~ynh1",
```
Comment (alexAubin) [inline manifest.json:9]: Eeeeh it sounds like we're moving backwards in terms of version number ?

Comment (ericgaspar) [reply manifest.json:9]: Damn yes... We went from alpha (1.0.0-alpha.80) to beta (1.0.0-beta.5) ...

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
(LGTM but didnt re-review since last review)

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**